libraries/AP_Scheduler/AP_Scheduler.tpp

```diff
@@ -0,0 +1,186 @@
+extern const AP_HAL::HAL& hal;
```

Comment:
The *.tpp file is part of the header as it gets included. However, I could include the tridge part again in the *.h.